Library/Formula/libantlr3c.rb

```diff
@@ -1,9 +1,12 @@
-require 'formula'
-
 class Libantlr3c < Formula
-  homepage 'http://www.antlr3.org'
-  url 'http://www.antlr3.org/download/C/libantlr3c-3.4.tar.gz'
-  sha1 'faa9ab43ab4d3774f015471c3f011cc247df6a18'
+  homepage "http://www.antlr3.org"
+  url "http://www.antlr3.org/download/C/libantlr3c-3.4.tar.gz"
+  sha1 "faa9ab43ab4d3774f015471c3f011cc247df6a18"
+  revision 1
+
+  option "with-exceptions", "Compile with support for exception handling"
+  option "with-antlrdebug", "Compile with antlr debugger in runtime"
+  option "without-shared", "Do not compile shared libraries"
```

Comment:
Pretty :-1: on this; presumably it's to build static libraries. Can the other formula not link dynamically instead?